src/main/cli/settings.c

```diff
@@ -208,6 +208,10 @@ static const char * const lookupTableGPSSBASMode[] = {
 static const char * const lookupTableGPSUBLOXModels[] = {
     "PORTABLE", "STATIONARY", "PEDESTRIAN", "AUTOMOTIVE", "AT_SEA", "AIRBORNE_1G", "AIRBORNE_2G", "AIRBORNE_4G"
 };
+
+static const char * const lookupTableGPSUBLOXUTCStandard[] = {
```

Comment:
Camel case
```suggestion
static const char * const lookupTableGpsUbloxUtcStandard[] = {
```

Reply:
i followed the other namings, so should we change all?

Reply:
I don't know about other places, but in camelCase only first must be capitalized, if not, methods that start with `GPSUBLOXUTC` will end in `gPSUBLOXUTC()` that is ugly, far better `gpsUbloxUtc()`. I suggest that at least for new methods, variables, etc. we do it in the correct way.

Reply:
This has been fixed.